fvm/state/state_holder.go

```diff
@@ -0,0 +1,41 @@
+package state
+
+// StateHolder provides active states
+// and facilitates common state management operations
+// in order to make services such as accounts not worry about
+// the state it is recommended that such services wraps
+// an state manager instead of an state itself.
+type StateHolder struct {
+	startState  *State
+	activeState *State
+	parents     map[*State]*State
```

Comment:
This isn't used anywhere, probably could be removed.

Reply:
nice catch, cleaned it.